charts/prefect-worker/templates/deployment.yaml

```diff
@@ -79,7 +79,7 @@ spec:
           env:
             - name: HOME
               value: /home/prefect
-            - name: PREFECT_AGENT_PREFETCH_SECONDS
+            - name: PREFECT_WORKER_PREFETCH_SECONDS
```

Comment:
Thanks for catching this bug and opening a pull request! Would you mind also updating the following variable from `PREFECT_AGENT_QUERY_INTERVAL` to `PREFECT_WORKER_QUERY_SECONDS`?

https://github.com/PrefectHQ/prefect/blob/9c6b870438d9c919db0625ca562b84b53d947ebd/src/prefect/settings.py#L1213-L1222

Reply:
Sorry for the delay, I didn't see your comment! It's done